pkg/controller/perconaservermongodb/ssl.go

```diff
@@ -209,6 +211,23 @@ func (r *ReconcilePerconaServerMongoDB) createSSLManualy(cr *api.PerconaServerMo
 	return nil
 }
 
+func getShardingSans(cr *api.PerconaServerMongoDB) []string {
```

Comment:
Not sure, but maybe we should check if sharding is enabled.

Reply:
as we discussed, we always add this since sharding can be enabled later

Reply:
yes, we add them always to not REgenerate when somebody enables sharding

Reply:
then instead of `cr.Spec.Sharding.ConfigsvrReplSet.Name` use `api.ConfigReplSetName`

Reply:
fixed in https://github.com/percona/percona-server-mongodb-operator/pull/485/commits/4e5cc3047aa62b895b7e1a41b26fc02bfbe2a526